### ISSUE TITLE
New Feature - Image search using captions

### ISSUE THREAD

**Issue by qwireq** [sukeesh/Jarvis#438]
The idea is to create captions for images in the given directory. This could be an interesting feature for Personal Assistant. Additionally, in the future, it also could be transformed into a search of images by a given description. Most images are stored with names, which are not connected with objects depicted on them. Therefore, it could be very useful functionality. 

I am planning to use a pre-trained deep learning model for image captioning. Of course, some of the captions could be complete nonsense, which could affect the accuracy of the search. However, still, the rest would be appropriate. If these features are interesting for you, I could proceed to development. Thanks.

**Comment by madhavkulkarni1986** [comment]
What deep learning model do you plan to use?

**Comment by qwireq** [comment]
> What deep learning model do you plan to use?

I am planning to use the model from [this tutorial](https://github.com/yunjey/pytorch-tutorial/tree/master/tutorials/03-advanced/image_captioning).

**Comment by pnhofmann** [comment]
close as stale
